tracee-ebpf/tracee/events_pipeline.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/aquasecurity/tracee/tracee-ebpf/external"
 )
 
-func (t *Tracee) runEventPipeline(done <-chan struct{}) error {
```

Comment:
I'm not sure it's a better design, to make the done channel communicate stats to the caller. done should trigger a graceful shutdown as quickly and cleanly as possible. now every place that wants to be able to gracefully shut down (this comment applies to the different stages of tracee) needs to have access to stats, and do the work to return them (will probably be more complex in the future). Instead, the caller can just call get stats after shutdown, which makes more sense IMO

Reply:
Good point. Let me see how I can improve that

Reply:
See #953